PodcastGenerator/themes/default/navbar.php

```diff
@@ -1,25 +1,27 @@
 <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
-    <a class="navbar-brand" href="<?php echo $config['indexfile']; ?>"><?php echo $config["podcast_title"]; ?></a>
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-    </button>
-    <div class="collapse navbar-collapse" id="navbarNav">
-        <ul class="navbar-nav">
-            <li class="nav-item">
-                <a class="nav-link" href="<?php echo $config['indexfile']; ?>">Home</a>
-            </li>
-            <?php
-            if (strtolower($config["categoriesenabled"]) == "yes") {
-            ?>
+    <div class="container">
+        <a class="navbar-brand" href="<?php echo $config['indexfile']; ?>"><?php echo $config["podcast_title"]; ?></a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
+            <ul class="navbar-nav">
                 <li class="nav-item">
-                    <a class="nav-link" href="categories.php"><?php echo $categories; ?></a>
+                    <a class="nav-link" href="<?php echo $config['indexfile']; ?>">Home</a>
```

Comment:
Was it intentional that we are changing the categories page link to the home page?

Reply:
> Was it intentional that we are changing the categories page link to the home page?

i dont remember making this change... it doesnt make sense to me...

Reply:
I strugle to find this code, i dont have it in my navbar.php (i guess i did something wrong 👎 )  how can i fix this?